src/cli/cli.cpp

```diff
@@ -540,30 +516,52 @@ template <> otError Interpreter::Process<Cmd("ba")>(Arg aArgs[])
 #if OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE
     else if (aArgs[0] == "ephemeralkey")
     {
+        bool enable;
+
         /**
          * @cli ba ephemeralkey
          * @code
          * ba ephemeralkey
-         * active
+         * Stopped
          * Done
          * @endcode
          * @par api_copy
-         * #otBorderAgentIsEphemeralKeyActive
+         * #otBorderAgentEphemeralKeyGetState
          */
         if (aArgs[1].IsEmpty())
         {
-            OutputLine("%sactive", otBorderAgentIsEphemeralKeyActive(GetInstancePtr()) ? "" : "in");
+            otBorderAgentEphemeralKeyState state = otBorderAgentEphemeralKeyGetState(GetInstancePtr());
+
+            OutputLine("%s", otBorderAgentEphemeralKeyStateToString(state));
         }
         /**
-         * @cli ba ephemeralkey set <keystring> [timeout-in-msec] [port]
+         * @cli ba ephemeralkey (enable, disable)
          * @code
-         * ba ephemeralkey set Z10X20g3J15w1000P60m16 5000 1234
+         * ba ephemeralkey
+         * Enabled
+         * Done
+         * @endcode
+         * @code
+         * ba ephemeralkey enable
+         * Done
+         * @endcode
+         * @cparam ba ephemeralkey [@ca{enable|disable}]
```

Comment:
For this one it seems that you have to use enable or disable right?  Since you separated out `ba ephemeralkey`.  So you could remove the optional brackets [ ]

Alternatively, combine this with `ba ephemeralkey` and make it the params optional

Reply:
Removed the `[ ]` for this.

Fixed all other comments in the new push. Thanks.